src/backend/cgxmm.c

```diff
@@ -1365,6 +1379,28 @@ void checkSetVex(code *c, tym_t ty)
             vreg |= 8;
         switch (c->Iop)
         {
+            case LODSS:
+            case LODSD:
+            case STOSS:
+            case STOSD:
+                if ((c->Irm & 0xC0) == 0xC0)
+                    break;
+            case LODAPS:
+            case LODUPS:
+            case LODAPD:
+            case LODUPD:
+            case LODDQA:
+            case LODDQU:
+            case LODD:
+            case LODQ:
+            case STOAPS:
+            case STOUPS:
+            case STOAPD:
+            case STOUPD:
+            case STODQA:
+            case STODQU:
+            case STOD:
+            case STOQ:
```

Comment:
Every time I try to generalize about the x86 instruction set, Intel adds more instructions that break it with more special cases :-(